libs/ml/include/ml/model/model.hpp

```diff
@@ -113,6 +113,12 @@ class Model
   friend struct serializers::MapSerializer;
   friend class fetch::vm_modules::ml::model::ModelEstimator;
 
+  using ChargeAmount = uint64_t;  // TODO(VH): Move me to an external declaration header.
```

Comment:
don't do this; get ChargeAmount from wherever it's genuinely declared!
If this means importing from vm we naturally don't want to do that from the ML library, so it would be better to move ChargeAmount from VM to meta/types.hpp or something like that, and then have vm/vm-modules/ml all include that

Reply:
meta/types.hpp sounds just great!

Reply:
/libs/meta/include/meta/vm_types.hpp created for this purpose.